src/components/dialogs/GifSelect.tsx

```diff
@@ -54,13 +56,18 @@ export function GifSelectDialog({
       break
   }
 
+  const renderErrorBoundary = useCallback(
+    (error: any) => <DialogError details={error.toString()} />,
```

Comment:
This isn't safe because technically `error` itself could be `null` or `undefined`.

Reply:
I guess we have this broken in a few other places. I'll punt on fixing that for now.